query_service/core/shared.py

```diff
@@ -27,5 +32,166 @@ def __str__(self):
         return self.message
 
 def convert_to_turtle(jsonlddata):
+        return Graph().parse(data=jsonlddata, format='json-ld').serialize(format="turtle")
 
-        return Graph().parse(data=jsonlddata, format='json-ld').serialize(format="turtle")
\ No newline at end of file
+
+def read_yaml_config(source_path: str) -> Dict[str, Any]:
```

Comment:
Take a peek at `pyyaml` just as an FYI for code like this

Reply:
Code updated (Commit-[9fc839c84](https://github.com/sensein/BrainKB/pull/8/commits/325cd88e4376aee4335fcb7ea3df1cf9fc839c84))